content/docs/observability/deployment/offline.md

```diff
@@ -250,7 +250,7 @@ To perform an offline installation of a Helm chart, the following steps should b
     kubectl get configmap powermax-config-params -n [CSI_DRIVER_NAMESPACE] -o yaml | sed 's/namespace: [CSI_DRIVER_NAMESPACE]/namespace: [CSM_NAMESPACE]/' | kubectl create -f -
     ```
 
-    If the CSI driver configmap name is not the default `powermax-config-params`, please use the following command to copy configmap:  
+    If the CSI driver configmap name is not the default `powermax-config-params`, please use the following command to copy configmap:
```

Comment:
... use the following command ...

(remove "please"

Reply:
Done